packages/ui-components/src/components/UIContextualMenu/UIContextualMenu.tsx

```diff
@@ -5,7 +5,7 @@ import type {
     ICalloutContentStyles,
     IRawStyle,
     IStyleFunctionOrObject,
-    ICalloutContentStyleProps
+    IContextualMenuStyleProps
```

Comment:
@Jimmy-Joseph19 Is this OK?

Reply:
This changes looks fine. Sorry for delay in response